src/views/product-landing/index.tsx

```diff
@@ -1,74 +1,44 @@
 /* eslint-disable react/no-array-index-key */
 import React, { ReactElement } from 'react'
-import slugify from 'slugify'
 import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
-import Heading, { HeadingProps } from 'components/heading'
-import Text from 'components/text'
-import DevDotContent from 'components/dev-dot-content'
-import GetStarted, { GetStartedProps } from './components/get-started'
-import Cards, { CardProps } from './components/cards'
+import { ProductLandingContent, ProductLandingContentBlock } from './schema'
 import s from './product-landing.module.css'
 
-type Block =
-  | ({ type: 'heading' } & HeadingProps & { heading: string })
-  | ({ type: 'get_started' } & GetStartedProps)
-  | ({ type: 'cards' } & CardProps)
-
-interface ProductLandingProps {
-  content: {
-    heading: string
-    subheading: string
-    blocks: Block[]
-  }
+/**
+ * TODO: make view-specific types as components are built out
+ * (authored content structure and content structure passed to the client
+ * will differ; eg tutorial card data will be filled in for client)
+ */
+interface ProductLandingViewProps {
+  content: ProductLandingContent
 }
 
-function ProductLandingView({ content }: ProductLandingProps): ReactElement {
+function ProductLandingView({
+  content,
+}: ProductLandingViewProps): ReactElement {
   return (
-    <DevDotContent>
-      <Heading
-        className={s.pageHeading}
-        size={500}
-        level={1}
-        slug={slugify(content.heading)}
-        weight="bold"
-      >
-        {content.heading}
-      </Heading>
-      {content.subheading && (
-        <Text className={s.pageSubheading}>{content.subheading}</Text>
-      )}
-      {content.blocks.map((block, idx) => {
+    <div>
+      <pre className={s.placeholder}>
+        <code>
+          {JSON.stringify({ ...content, blocks: undefined }, null, 2)}
+        </code>
+      </pre>
+      {content.blocks.map((block: ProductLandingContentBlock, idx: number) => {
         const { type } = block
-        if (type === 'heading') {
-          const { heading, slug, level, size } = block
-          return (
-            <Heading
-              className={s[`h${level}`]}
-              key={idx}
-              weight="bold"
-              {...{ slug, level, size }}
-            >
-              {heading}
-            </Heading>
-          )
-        } else if (type === 'get_started') {
-          const { heading, product, text, link } = block
-          return <GetStarted key={idx} {...{ product, heading, text, link }} />
-        } else if (type === 'cards') {
-          const { columns, cards } = block
-          return <Cards key={idx} {...{ columns, cards }} />
+        switch (type) {
```

Comment:
In short, I've replaced the view with a bunch of `<pre />` blocks that show:
1. the data
    - ... which right now is not sufficient to actually render the page; it needs to be augmented server-side. Intent is to do this component-by-component in subsequent PRs.
2. the component breakdown
    - ... which right now is based on the block `type`; each component will eventually have a separate home in `views/product-landing/components`. Intent is to build these in subsequent PRs.